requirements-agent-release.txt

```diff
@@ -44,7 +44,7 @@ datadog-ibm-db2==1.1.0
 datadog-ibm-mq==2.0.0; sys_platform == 'linux2'
 datadog-ibm-was==1.1.0
 datadog-iis==2.6.0; sys_platform == 'win32'
-datadog-istio==2.3.0
+datadog-istio==2.4.0
```

Comment:
Another release related change, please remove this.

Reply:
Done